### PR TITLE
Ansible fixes

### PR DESCRIPTION
**Pull request by Nonsense331**

### CODE REVIEW

File: ansible/roles/common/tasks/main.yml [hunk 1]
```diff
@@ -64,6 +64,13 @@
     tags:
       - maintenance
 
+  - name: apt-get update
+    apt: update_cache=yes cache_valid_time=86400
+    become: true
+    tags:
+      - upgrade
+      - maintenance
+
```
Comment (petitJAM) [inline ansible/roles/common/tasks/main.yml:73]: Seems like this should be _after_ `install aptitude`. Why'd you move it?

Comment (Nonsense331) [reply ansible/roles/common/tasks/main.yml:73]: oh i think i needed it before the nginx thing and moved it up too far?

**Comment by petitJAM** [comment]
Don't forget the changelog entry

File: ansible/roles/nginx-rails/tasks/main.yml [hunk 2]
```diff
@@ -9,7 +9,15 @@
   file:
     src: /etc/nginx/sites-available/{{project_name}}
     dest: /etc/nginx/sites-enabled/{{project_name}}
-    state: "{{ (ssl_enabled and nginx_ssl_config is defined) | ternary('absent', 'link') }}"
+    state: link
+  when: ssl_enabled != true or nginx_ssl_config is not defined
+  become: true
```
Comment (samsonasu) [inline ansible/roles/nginx-rails/tasks/main.yml:14]: I think the old code removed it if ssl was no longer enabled which this won't do if I'm reading it correctly

Comment (samsonasu) [reply ansible/roles/nginx-rails/tasks/main.yml:14]: ok wait actually i think this block and the next block are switched

**Comment by samsonasu** [comment]
:shipit: once you're sure it works

**Pull request merged**